meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -7,7 +7,7 @@ DEPENDS_class-native = ""
 
 PACKAGES_append = " mender-modules-gen"
 
-RDEPENDS_${PN}_append_mender-growfs-data_mender-systemd = " parted util-linux-fdisk"
+RDEPENDS_${PN}_append_mender-growfs-data_mender-systemd = " parted util-linux-fdisk util-linux-blkid"
```

Comment:
This dependency should be added only if `mender-partlabel` is enabled. Try with:
```suggestion
RDEPENDS_${PN}_append_mender-growfs-data_mender-systemd = " parted util-linux-fdisk"
RDEPENDS_${PN}_append_mender-growfs-data_mender-systemd_mender-partlabel = " util-linux-blkid"
```